### ISSUE TITLE
painterResource is too slow on iOS

### ISSUE THREAD

**Issue by paxbun** [icerockdev/moko-resources#526]
The MOKO resources version of `painterResource` is much slower than its JetBrains counterpart. For example,
```kotlin
LazyVerticalGrid {
    repeat(100) {
        item {
            // Too slow, the app freezes as the screen is scrolled
            Image(painterResource(MR.images.someImage))
            // Ok
            Image(painterResource("MR/images/someImage@3x.png"))
        }
    }
}
```

It seems like retrieving images through UIImage, CGImage, and Skia Image conversions is too slow; why don't we just use the same strategy as in the JVM version (although the `resource` API is experimental)?

**Comment by paxbun** [comment]
We modified MOKO resources as follows, and it works well.
https://github.com/clebrain/moko-resources/blob/2cedf849eadd4be7f86ff7fa4ba2d6606bf15aa9/resources-compose/src/iosMain/kotlin/dev/icerock/moko/resources/compose/ImageResource.kt#L22-L59

**Comment by DasJhaman** [comment]
Hi @Alex009 is this has been released? I am also facing a similar issue on iOS.

**Comment by Alex009** [comment]
@DasJhaman no. it's planned in 0.24.0

**Comment by HahhVwgwg** [comment]
@Alex009 Could you please provide an estimated timeframe for the release of version **0.24.0**?

**Comment by pcorreia89** [comment]
Hi @Alex009, I was using 0.23.0 version and got the same issue with the painterResource.

I've tried to upgrade to 0.24.0-alpha04 but I am getting the following when trying to load something from MR.images:

Uncaught Kotlin exception: kotlin.native.internal.FileFailedToInitializeException: There was an error during file or class initialization
    at 0   appIos                              0x105643b87        kfun:kotlin.Error#<init>(kotlin.String?;kotlin.Throwable?){} + 143 
    at 1   appIos                              0x105680e07        kfun:kotlin.native.internal.FileFailedToInitializeException#<init>(kotlin.String?;kotlin.Throwable?){} + 143 
    at 2   appIos                              0x105680063        ThrowFileFailedToInitializeException + 275 
    at 3   appIos                              0x1057f5623        CallInitGlobalPossiblyLock + 759 
    at 4   appIos                              0x104fa4503        kfun:studio.lostjoker.smartdealer.MR.images#<get-$instance>#static(){}studio.lostjoker.smartdealer.MR.images + 75 
    at 5   appIos                              0x104ff335f       

I've followed all the steps (replace resources to moko-resources, etc.).

Any idea what it might be?

Thank you very much.

**Comment by pcorreia89** [comment]
@Alex009 Could you please provide an estimated timeframe for the release of version 0.24.0?

**Comment by Alex009** [comment]
fixed in 0.24.0-beta-1
